libs/accounts/src/lib/accounts-table.tsx

```diff
@@ -1,340 +1,187 @@
-    onClickTransfer,
-    isReadOnly,
-    showDepositButton,
-    t,
-  ]);
+          )}
+        </div>
+      </div>
+      <ProgressBar compact value={value} intent={intent} />
+    </div>
+  );
+};
 
-  const data = rowData?.filter((data) => data.asset.id !== pinnedAsset?.id);
+export const AccountCard = ({
+  asset,
+  isReadOnly,
+  partyId,
+  ...actions
+}: {
+  asset: AssetFieldsFragment;
+  isReadOnly?: boolean;
+  partyId?: string;
+} & Actions) => {
+  const t = useT();
+  const [expanded, setExpanded] = useState(false);
+  const { data } = useDataProvider({
+    dataProvider: aggregatedAccountDataProvider,
+    variables: { partyId: partyId || '', assetId: asset.id },
+    skip: !partyId,
+  });
+  return (
+    <div className="p-3 border-b border-default relative">
+      <div className="flex items-center mb-3">
+        <Emblem asset={asset.id} />
+        <span className="grow ml-2 text-lg">{asset.symbol}</span>
+        <div className="z-10">
+          <AccountsActionsDropdown
+            isReadOnly={isReadOnly}
+            assetId={asset.id}
+            assetContractAddress={
+              asset.source?.__typename === 'ERC20'
+                ? asset.source.contractAddress
+                : undefined
+            }
+            onClickDeposit={() => {
+              actions.onClickDeposit?.(asset.id);
+            }}
+            onClickWithdraw={() => {
+              actions.onClickWithdraw?.(asset.id);
+            }}
+            onClickTransfer={() => {
+              actions.onClickTransfer?.(asset.id);
+            }}
+          />
+        </div>
+      </div>
+      {data?.breakdown?.map((data) => (
+        <BreakdownItem data={data} />
+      ))}
+      <div className="flex items-center mt-3">
+        <div className="text-base font-alpha">{t('Total')}</div>
+        <div className="text-right leading-tight font-alpha grow text-base">
+          {addDecimalsFormatNumberQuantum(
+            data?.total || '0',
+            asset.decimals,
+            asset.quantum
+          )}
+        </div>
+      </div>
+      {expanded ? (
+        <div className="grid gap-1 grid-cols-4 mt-3">
+          <Button
+            onClick={() => actions.onClickDeposit?.(asset.id)}
+            label={t('Deposit')}
+            icon={VegaIconNames.DEPOSIT}
+          />
+          <Button
+            onClick={() => actions.onClickSwap?.(asset.id)}
+            label={t('Swap')}
+            icon={VegaIconNames.SWAP}
+          />
+          <Button
+            onClick={() => actions.onClickTransfer?.(asset.id)}
+            label={t('Transfer')}
+            icon={VegaIconNames.TRANSFER}
+          />
+          <Button
+            onClick={() => actions.onClickWithdraw?.(asset.id)}
+            label={t('Withdraw')}
+            icon={VegaIconNames.WITHDRAW}
+          />
+        </div>
+      ) : (
+        <button
+          className="absolute inset-0"
+          onClick={() => setExpanded(true)}
+        ></button>
```

Comment:
Lets add some content inside the button fir a11y `<span className="sr-only">Show asset actions</div>`